logging/bin/deploy_logging_open_openshift.sh

```diff
@@ -38,90 +38,87 @@ log_notice "Deploying logging components to the [$LOG_NS] namespace [$(date)]"
 
 
 ##################################
-# OpenShift Set-up               #
+# OpenShift-specific Set-up      #
 ##################################
-
-# OpenShift-specific set-up/config
 log_info "STEP 0: OpenShift Setup"
 logging/bin/deploy_openshift_prereqs.sh
 
+# On OpenShift force "full" TLS
+export LOG_KB_TLS_ENABLE=true
+
 ##################################
 # Event Router                   #
 ##################################
-
-# Collect Kubernetes events as pseudo-log messages?
-
 log_info "STEP 1: Event Router"
 logging/bin/deploy_eventrouter.sh
 
 
 ##################################
 # Open Distro for Elasticsearch  #
 ##################################
-
 log_info "STEP 2: Elasticsearch"
 logging/bin/deploy_elasticsearch_open.sh
 
 
 ##################################
 # ODFE Content (OpenShift)       #
 ##################################
-
 log_info "STEP 2a: Loading Content into Elasticsearch"
 logging/bin/deploy_elasticsearch_content_open_openshift.sh
 
 
 ##################################
 # Elasticsearch Metric Exporter  #
 ##################################
-
 log_info "STEP 3: Elasticsearch metric exporter"
 logging/bin/deploy_esexporter.sh
 
 
 ##################################
-# Create OpenShift Routes        #
+# Create OpenShift Route(s)      #
+# and display app URL(s)         #
 ##################################
-log_info "STEP 4a: Create OpenShift Route(s)"
-logging/bin/deploy_openshift_routes.sh
+log_info "STEP 4a: Create OpenShift Route(s) & Application URL(s)"
 
-##################################
-# Kibana Content                 #
-##################################
+OPENSHIFT_ROUTES_ENABLE=${OPENSHIFT_ROUTES_ENABLE:-true}
 
-# NOTE: For ODFE, Kibana is deployed as part of ES Helm chart
+if [ "$OPENSHIFT_ROUTES_ENABLE" == "true" ]; then
 
-log_info "STEP 4b: Configuring Kibana"
+   servicelist="KIBANA"
+   logging/bin/create_openshift_route.sh KIBANA
 
-export KB_KNOWN_NODEPORT_ENABLE=false
-logging/bin/deploy_kibana_content.sh
+   ES_ENDPOINT_ENABLE=${ES_ENDPOINT_ENABLE:-false}
+   if [ "$ES_ENDPOINT_ENABLE" == "true" ]; then
```

Comment:
`ES_ENDPOINT_ENABLE` should probably be `OPENSHIFT_ES_ROUTE_ENABLE` or something similar.

Reply:
I started with that but I'd like to support the env var in non-OpenShift environments at some point.  In the sample user.env files, the comments indicate that it is OpenShift specific.

Reply:
Well, "endpoint" is the wrong term to use here regardless. Kubernetes' defines an 'endpoint' as one target that a service resolves to. They've since moved to EndpointSlices, but still, let's avoid the use of 'endpoint'.

IMO, if we have `OPENSHIFT_ROUTES_ENABLE`, we should have `OPENSHIFT_ES_ROUTE_ENABLE`.

I think OpenShift is just different. We can have INGRESS_ENABLE in the future for non-OpenShift environments.

Reply:
Check latest push...changed ES_ENDPOINT_ENABLE to OPENSHIFT_ES_ROUTE_ENABLE in code and two user.env files.